roles/edpm_nodes_validation/tasks/main.yml

```diff
@@ -67,7 +67,7 @@
   ansible.builtin.shell: |
     HOSTNAME=$(hostname)
     SHORT_NAME=$(hostname -s)
-    FQDN_FROM_HOSTS=$(awk '$3 == "'${SHORT_NAME}'"{print $2}' /etc/hosts)
+    FQDN_FROM_HOSTS=$(awk '$1 !~ /#/ && $3 == "'${SHORT_NAME}'"{print $2}' /etc/hosts)
```

Comment:
OK so we are just ignoring the commented out lines

Reply:
this is now running in the molecue test 
[
Verify the configured FQDN vs /etc/hosts](https://github.com/openstack-k8s-operators/edpm-ansible/actions/runs/8346229798/job/22843049848#step:7:500)


```
TASK [osp.edpm.edpm_nodes_validation : Verify the configured FQDN vs /etc/hosts] ***
  task path: /home/runner/.cache/ansible-compat/63fb77/collections/ansible_collections/osp/edpm/roles/edpm_nodes_validation/tasks/main.yml:66
  redirecting (type: connection) ansible.builtin.podman to containers.podman.podman
  Loading collection containers.podman from /home/runner/.cache/ansible-compat/63fb77/collections/ansible_collections/containers/podman
  Using podman connection from collection
  <instance> RUN [b'/usr/bin/podman', b'mount', b'instance']
  Failed to mount container instance: b'Error: cannot run command "podman mount" in rootless mode, must execute `podman unshare` first'
  <instance> RUN [b'/usr/bin/podman', b'exec', b'instance', b'/bin/sh', b'-c', b'echo ~ && sleep 0']
  <instance> RUN [b'/usr/bin/podman', b'exec', b'instance', b'/bin/sh', b'-c', b'( umask 77 && mkdir -p "` echo /root/.ansible/tmp `"&& mkdir "` echo /root/.ansible/tmp/ansible-tmp-1710938452.6168222-11005-254617367915219 `" && echo ansible-tmp-1710938452.6168222-11005-254617367915219="` echo /root/.ansible/tmp/ansible-tmp-1710938452.6168222-11005-254617367915219 `" ) && sleep 0']
  Using module file /opt/hostedtoolcache/Python/3.9.18/x64/lib/python3.9/site-packages/ansible/modules/command.py
  <instance> PUT /home/runner/.ansible/tmp/ansible-local-9410xatbxk5h/tmp3hvssntq TO /root/.ansible/tmp/ansible-tmp-1710938452.6168222-11005-254617367915219/AnsiballZ_command.py
  <instance> RUN [b'/usr/bin/podman', b'cp', b'/home/runner/.ansible/tmp/ansible-local-9410xatbxk5h/tmp3hvssntq', b'instance:/root/.ansible/tmp/ansible-tmp-1710938452.6168222-11005-254617367915219/AnsiballZ_command.py']
  <instance> RUN [b'/usr/bin/podman', b'exec', b'instance', b'/bin/sh', b'-c', b'chmod u+x /root/.ansible/tmp/ansible-tmp-1710938452.6168222-11005-254617367915219/ /root/.ansible/tmp/ansible-tmp-1710938452.6168222-11005-254617367915219/AnsiballZ_command.py && sleep 0']
  <instance> RUN [b'/usr/bin/podman', b'exec', b'instance', b'/bin/sh', b'-c', b'/usr/bin/python3.9 /root/.ansible/tmp/ansible-tmp-1710938452.6168222-11005-254617367915219/AnsiballZ_command.py && sleep 0']
  <instance> RUN [b'/usr/bin/podman', b'exec', b'instance', b'/bin/sh', b'-c', b'rm -f -r /root/.ansible/tmp/ansible-tmp-1710938452.6168222-11005-254617367915219/ > /dev/null 2>&1 && sleep 0']
  ok: [instance] => {
      "changed": false,
      "cmd": "HOSTNAME=$(hostname)\nSHORT_NAME=$(hostname -s)\nFQDN_FROM_HOSTS=$(awk '$1 !~ /#/ && $3 == \"'${SHORT_NAME}'\"{print $2}' /etc/hosts)\nif [[ $HOSTNAME != $FQDN_FROM_HOSTS ]]; then\n  echo \"FAILURE\"\n  echo -e \"System hostname: ${HOSTNAME}\\nEntry from /etc/hosts: ${FQDN_FROM_HOSTS}\\n\"\n  exit 1\nfi\nexit 0\n",
      "delta": "0:00:00.007222",
      "end": "2024-03-20 12:40:53.293250",
      "invocation": {
          "module_args": {
              "_raw_params": "HOSTNAME=$(hostname)\nSHORT_NAME=$(hostname -s)\nFQDN_FROM_HOSTS=$(awk '$1 !~ /#/ && $3 == \"'${SHORT_NAME}'\"{print $2}' /etc/hosts)\nif [[ $HOSTNAME != $FQDN_FROM_HOSTS ]]; then\n  echo \"FAILURE\"\n  echo -e \"System hostname: ${HOSTNAME}\\nEntry from /etc/hosts: ${FQDN_FROM_HOSTS}\\n\"\n  exit 1\nfi\nexit 0\n",
              "_uses_shell": true,
              "argv": null,
              "chdir": null,
              "creates": null,
              "executable": null,
              "removes": null,
              "stdin": null,
              "stdin_add_newline": true,
              "strip_empty_ends": true
          }
      },
      "msg": "",
      "rc": 0,
      "start": "2024-03-20 12:40:53.286028",
      "stderr": "",
      "stderr_lines": [],
      "stdout": "",
      "stdout_lines": []
  }
```